src/app/history/history-panel.component.ts

```diff
@@ -66,4 +66,20 @@ export class HistoryPanelComponent extends GraphExplorerComponent implements OnI
             this.queryRunnerService.executeExplorerQuery(true);
         }
     }
+
+    public exportQuery(query: IGraphApiCall) {
+      const blob = new Blob([query.har], { type: 'text/json' });
+      const filename = 'graph-explorer.har';
```

Comment:
Since we only export a single call, let's set the filename to IGraphApiCall.humanName if available, or if not, base it on the query and replace the '/' with '_'